pdm/_vendor/halo/_utils.py

```diff
@@ -4,10 +4,7 @@
 import codecs
 import platform
 import six
-try:
```

Comment:
i *think* this is no longer required in python 3.7+?

i'll let the CI confirm that